components/Profile.js

```diff
@@ -0,0 +1,32 @@
+import { useContext } from 'react';
+import BlogContext from '../context/BlogContext';
+
+export default function Profile() {
+  const getContext = useContext(BlogContext);
+  const profiles = getContext.post.profiles
```

Comment:
where are these profiles coming from?

why not have this component support props and you just provide data through props?

Reply:
The profiles come from the post details...

---
title: "lorem ipsum"
profiles: "path/to/list/of/profiles
---

`All you have to do in the md is to use the profile component`

`<Profile />`

Reply:
> why not have this component support props and you just provide data through props?

If I Import the component in the markdown, how do i pass children in MD?

Reply:
I'm just afraid passing data to component through metadata might not be super intuitive as we do it classical way always, like:
```
<YouTube id="Ni5tCY9r0TY" />
```

so component with props

look how next/previous works:
```
<DocsButton suggestions={[ { href:'/docs/concepts/consumer', type: 'back',
title: 'Consumer', }, { href:'/docs/concepts/message', type: 'next',
title: 'Message', } ]} />
```

so IMHO `Profile` should work same way, you add component with array of profiles

Reply:
Thanks for the update @derberg. updated the PR